src/scratchpads/chat_utils_prompts.rs

```diff
@@ -56,6 +56,9 @@ async fn _workspace_info(workspace_dirs: &[String], active_file_path: &Option<Pa
     } else {
         info.push_str("\n\nThere is no active file with version control, complain to user if they want to use anything git/hg/svn/etc and ask to open a file in IDE for you to know which project is active.");
     }
+    if !background_tools_running.is_empty() {
+        info.push_str(&format!("\n\nThe tools that are currently in a background:\n{}", background_tools_running.join(", ")));
+    }
```

Comment:
The model already knows it started the process

Reply:
background tool is not bound to a chat lifetime.
Scenario:  you launch tool in one chat, close it, open another, tool is still alive. We have to inform chat about active tools